code/controllers/subsystem/processing/quirks.dm

```diff
@@ -68,6 +68,21 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 	if(cli.get_exp_living(TRUE) < EXP_ASSIGN_WAYFINDER && !user.has_quirk(/datum/quirk/needswayfinder))
 		user.add_quirk(/datum/quirk/needswayfinder, TRUE)
 
+	var/list/player_records = cli.prefs.exp
+	var/assistant_playtime = text2num(player_records["Assistant"])
+	var/bad_quirks_to_add = assistant_playtime % 100
```

Comment:
Playtime is tracked by minute, not hour, internally.

Reply:
fixed